PubNubUnity/Assets/PubNub/PlayModeTests/PlayModeCommon.cs

```diff
@@ -8,6 +8,7 @@
 namespace PubNubAPI.Tests
 {
 
+#if !UNITY_WSA && !UNITY_WSA_10_0
```

Comment:
Any reason why code is getting skipped for UNITY_WSA AND UNITY_WSA_10_0 ?

Reply:
Please review the known limitations [here](https://docs.unity3d.com/Packages/com.unity.test-framework@1.0/manual/index.html)